app/src/main/res/values-cy/strings.xml

```diff
@@ -690,17 +690,17 @@
     <string name="filter_description_format">%1$s: %2$s</string>
     <string name="post_media_image">Delwedd</string>
     <string name="select_list_manage">Rheoli rhestrau</string>
-    <string name="help_empty_home">Dyma\'ch <b>llinell amser cartref</b>. Mae\'n dangos negeseuon diweddar y cyfrifon rydych chi\'n eu dilyn. 
-\n 
+    <string name="help_empty_home">Dyma\'ch <b>llinell amser cartref</b>. Mae\'n dangos negeseuon diweddar y cyfrifon rydych chi\'n eu dilyn.
```

Comment:
Hm?

Weren't those the spaces that were added before - and nobody knew why?

Reply:
I think so. Well now we know, Android Studio formats stuff it shouldn't when renaming a string.

Reply:
`sed`to the rescue 🙄